### ISSUE TITLE
Slack emoji not being set

### ISSUE THREAD

**Issue by RyanMacG** [parkr/capistrano-slack-notify#13]
I have the below in my `deploy.rb` as per the readme. 

``` ruby
set :slack_username, "Deploys"
set :slack_room, "#deploys"
set :slack_emoji, ":shipit:"
set :deployer, ENV['USER'].capitalize
```

However the emoji being shown is the default webhook one, I've attached an image as an example.
![Example output](https://cloud.githubusercontent.com/assets/341231/6828967/8113d466-d308-11e4-9172-506909dff293.png)

**Comment by JonCrawford** [comment]
:+1: I thought I was doing something wrong and didn't feel like investigating.

**Comment by parkr** [comment]
Interesting! I'll get into it.

**Comment by parkr** [comment]
Released in v1.3.1
